builder/buildable.go

```diff
@@ -38,33 +38,28 @@ func NewBuildable(client client.Client,
 	}
 }
 
-func (buildable *Buildable) GetWorkflowBuild(req ctrl.Request, workflowID string) (apiv08.KogitoServerlessBuild, error) {
-	buildInstance := &apiv08.KogitoServerlessBuild{}
-	buildInstance.Spec.WorkflowId = workflowID
-	error := buildable.Client.Get(buildable.Ctx, req.NamespacedName, buildInstance)
-	return *buildInstance, error
+// GetWorkflowBuild gets the required Build associated with the same name/namespaced as defined in the request, nil if not found
```

Comment:
Why isn't used the unique ID ?

Reply:
Please see my comment in the PR, I won't touch this use case now.